vtex-sites/storecomponents.store
# Fixing cart-related pixel events

## What's the purpose of this pull request?
This PR fixes cart-related pixel events on the store. Add to cart buttons and quantity selectors should now emit rich events containing all the data needed for SAE and GTM to work properly. 

## How it works? 
`removeFromCart` events should be sent when user decreases the quantity or removes a product from the cart. Adding products to the card or increasing their quantity should emit `addToCart` events.

## How to test it?
Add products to the cart, change the quantities and remove items from the cart.

I use the following script to see the events on my console:

```js
window.addEventListener("message", function (event) {
  if (event.data?.data?.type?.startsWith("vtex:")) {
    console.log(event.data.data);
  }
});
```
## References

I'm pointing to temporary dependencies of the faststore packages, but I'll change them before deploying.
